src/OrchardCore.Modules/OrchardCore.Title/Drivers/TitlePartDisplay.cs

```diff
@@ -26,8 +34,7 @@ public override IDisplayResult Edit(TitlePart titlePart)
             {
                 model.Title = titlePart.ContentItem.DisplayText;
                 model.TitlePart = titlePart;
-
-                return new ValueTask();
+                model.Settings = GetSettings(titlePart);
```

Comment:
How come the ValueTask has been removed?
I'm not totally caught up on the change to all the overrides here, so just checking?

Reply:
I just looked at other drivers and they don't return anything. Maybe there is an explanation but my guess is that the return is not required.

Reply:
Ah, I see there's an override to the func that returns a value task when it's removed.
So yes, it probably didn't need to be there in the first place, as underlying in one of the Initialize it will do it

Reply:
And overriding the one that uses a `Func<TModel, ValueTask>` allows to do async calls in the delegate.